Libraries/Components/Button.windows.js

```diff
@@ -116,6 +143,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    var content =
+        <View style={buttonStyles}>
+          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
+        </View>;
+
+    if (Platform.OS === "windows") {
+      var tabIndex = this.props.tabIndex || 0;
```

Comment:
>tabIndex = this.props.tabIndex || 0 [](start = 10, length = 35)

What's the rationale for defaulting this to `0` instead of `undefined`?

Reply:
I followed the Web convention (that the ReactNative API kind of follows) to make the buttons tab-stoppable by default. Same is true for TextInput.

Reply:
I followed the Web convention (that the ReactNative API kind of follows) to make the buttons tab-stoppable by default. Same is true for TextInput.

---
In reply to: [156557254](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156557254) [](ancestors = 156557254)